aws/resource_aws_cloudwatch_event_target.go

```diff
@@ -175,11 +186,37 @@ func resourceAwsCloudWatchEventTarget() *schema.Resource {
 								},
 							},
 						},
+						"placement_constraints": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							MaxItems: 10,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"expression": {
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+									"type": {
+										Type:         schema.TypeString,
+										Required:     true,
+										ValidateFunc: validation.StringInSlice(events.PlacementConstraintType_Values(), false),
+									},
+								},
+							},
+						},
 						"platform_version": {
 							Type:         schema.TypeString,
 							Optional:     true,
 							ValidateFunc: validation.StringLenBetween(0, 1600),
 						},
+						"propagate_tags": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      events.PropagateTagsTaskDefinition,
+							ValidateFunc: validation.StringInSlice(events.PropagateTags_Values(), false),
+						},
+						"tags":     tagsSchema(),
+						"tags_all": tagsSchemaComputed(),
```

Comment:
I wouldn't have `tags_all` here as the `SetTagsDiff` `CustomizeDiff` function won't know anything about them and could cause errors.
Removing this attribute should simplify the implementation as there will be no assumption that default tags and ignore tags apply (can remove the added `meta` argument to various functions).

Reply:
removed default tagging here.